src/ansi-c/expr2c.cpp

```diff
@@ -41,6 +41,21 @@ Author: Daniel Kroening, kroening@kroening.com
 #include "expr2c.h"
 #include "expr2c_class.h"
 
+namespace {
+  std::string escape(const std::string &str)
+  {
+    std::string result;
+    result.reserve(str.size());
+    for(const auto &ch : str)
+    {
+      if(ch=='"')
```

Comment:
While we're here should we escape unprintable things (newlines, tabs, the weird ASCII control chars below 10) at the same time?